### PR TITLE
[stable/neo4j] Fix dataVolume Persistence

### PR DESCRIPTION
**Pull request by smelchior**
The current chart was not really persisting the data volume as `volumemounts` was defined twice and the second one for `plugins` was active. 
The data PVC was created but not mounted. This caused some data loss during our testing as we did run the Set with only 1 core server.
I also made the subPath optional to make the definition a little more readable.

### CODE REVIEW

File: stable/neo4j/README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ their default values.
 | `core.persistentVolume.storageClass` | Storage class of backing PVC                                                                                                            | `standard` (uses beta storage class annotation) |
 | `core.persistentVolume.size`         | Size of data volume                                                                                                                     | `10Gi`                                          |
 | `core.persistentVolume.mountPath`    | Persistent Volume mount root path                                                                                                       | `/data`                                         |
+| `core.persistentVolume.subPath`    | Subdirectory of the volume to mount                                                                                                       | `nil`                                           |
```
Comment (unguiculus) [inline stable/neo4j/README.md]: Proper alignment, please.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. I have only one nit.

**Comment by unguiculus** [comment]
@mneedham

**Comment by smelchior** [comment]
whoops, yes indeed. Fixed now :-)

**Comment by smelchior** [comment]
ping @unguiculus

File: stable/neo4j/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: neo4j
 home: https://www.neo4j.com
-version: 0.5.0
+version: 0.5.1
```
Comment (timstoop) [inline stable/neo4j/Chart.yaml]: As you're adding a feature, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by timstoop** [comment]
Can you please rebase and check the version in Chart again?

**Comment by smelchior** [comment]
sorry, did not see that, it is done now

**Review by unguiculus — APPROVED** [review]

**Pull request merged**